lib/presentation/pages/home/partial/current_match_widget.dart

```diff
@@ -0,0 +1,185 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:liga_inggris_mobile/app/config/app_colors.dart';
+import 'package:liga_inggris_mobile/presentation/common/clickable_card_widget.dart';
+import 'package:liga_inggris_mobile/presentation/common/club_logo_widget.dart';
+import 'package:liga_inggris_mobile/presentation/controllers/home/home_controller.dart';
+import 'package:liga_inggris_mobile/services/match/model.dart';
+
+class CurrentMatchWidget extends GetView<HomeController> {
+  final RxList<MatchModel> currentMatch;
+
+  const CurrentMatchWidget({
+    super.key,
+    required this.currentMatch,
+  });
+
+  void _handleTapCard(int matchId) {
+    controller.goToMatchDetail(matchId);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    if (currentMatch.isEmpty) {
+      return const Column(
+        children: [
+          SizedBox(height: 10),
+          Text(
+            "Pertandingan Saat Ini",
+            style: TextStyle(fontSize: 24),
+          ),
+          Center(
+            child: Text(
+              "Tidak ada pertandingan",
+              style: TextStyle(fontSize: 16),
+            ),
+          ),
+        ],
+      );
+    }
+    final match = currentMatch.first;
```

Comment:
```suggestion
    }
    
    final match = currentMatch.first;
    
```
dikasih space biar enak dibaco